AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java

```diff
@@ -143,7 +143,17 @@ private void login() {
 
         String username = mUsername.getText().toString().trim(); // trim spaces, issue 1586
         String password = mPassword.getText().toString();
+        if(username.isEmpty()) {
```

Comment:
Spacing - I won't comment the other occurrences.
```suggestion
        if (username.isEmpty()) {
```